dashboard/src/components/AppView/AppView.test.tsx

```diff
@@ -1,4 +1,5 @@
 import actions from "actions";
+import { getType } from "typesafe-actions";
```

Comment:
I've refactored a little in this file because the existing tests aren't testing that actions are dispatched by the view, but rather that the actions themselves are called... this meant that my test passed even though I'd missed `dispatch`ing the action. So I've left the existing tests as they are but updated the ones for the functionality that I'm changing so that they actually assert that the correct action is dispatched. The other issue I had from what I'd included in the diff yesterday was that I hadn't enabled the resources plugin :)

Reply:
Thanks for the info and refactoring the test. I've just hit the same issue... the error message wasn't clear at all so, almost running out of ideas I decided to check if the resources plugin was enabled... :facepalm: